learn-responsive-web-design-by-building-a-piano/index08.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Responsive Piano</title>
+</head>
+
+<style>
+
+</style>
+
+<body>
+
+    <div id="piano">
+
+        <div class="keys">
+            <div class="key"></div>
+            <div class="key black--key"></div>
+            <div class="key black--key"></div>
+            <div class="key"></div>
+            <div class="key black--key"></div>
+            <div class="key black--key"></div>
+            <div class="key black--key"></div>
+
+            <div class="key"></div>
+            <div class="key black--key"></div>
+            <div class="key black--key"></div>
+            <div class="key"></div>
+            <div class="key black--key"></div>
+            <div class="key black--key"></div>
+            <div class="key black--key"></div>
+
+            <div class="key"></div>
+            <div class="key black--key"></div>
+            <div class="key black--key"></div>
+            <div class="key"></div>
+            <div class="key black--key"></div>
+            <div class="key black--key"></div>
+            <div class="key black--key"></div>
+        </div>
+    </div>
+ 
+</body>
+</html>
+
+<!-- 
+    Before giving style or form to the piano, you must reset the
```

Comment:
This will also have to be broken down quite a bit. The first lesson can be to create a rule targeting `html` and setting `box-sizing` to `border-box`.

The next can be a brief review of `*` and pseudo elements, which students have only seen once or twice before this, and setting them to `box-sizing: inherit;`.